src/snackbar/simple-snackbar.element.ts

```diff
@@ -0,0 +1,83 @@
+import { customElement, bindable, Animator, ComponentAttached, ComponentDetached } from "aurelia-templating";
+import { autoinject } from "aurelia-dependency-injection";
+import { LoggerFactory, ILog } from "@ssv/au-core";
+
+import { SnackbarRef } from "./snackbar-ref";
+import { snackbarConfig } from "./snackbar.config";
+
+const PREFIX = "ssv-simple-snackbar";
+const ACTIVE_CLASS = `${PREFIX}--active`;
+
+@autoinject()
+@customElement(PREFIX)
+export class SimpleSnackbarElement implements ComponentAttached, ComponentDetached {
+
+	@bindable snackbarRef: SnackbarRef;
+
+	label: string;
+	action: string | undefined;
+	actionColor = snackbarConfig.actionColor;
+	actionType = snackbarConfig.actionType;
+
+	private logger: ILog;
+	private timeoutToken: number | undefined;
+	private isAttached = false;
+
+	constructor(
+		loggerFactory: LoggerFactory,
+		private element: Element,
+		private animator: Animator
+	) {
+		this.logger = loggerFactory.get("simpleSnackbarElement");
+	}
+
+	attached() {
+		this.isAttached = true;
+		this.handleChange();
+	}
+
+	detached() {
+		this.isAttached = false;
+	}
+
+	snackbarRefChanged() {
+		if (this.isAttached) {
+			this.handleChange();
+		}
+	}
+
+	onAction() {
+		this.logger.debug("onAction");
```

Comment:
if you dont need all these `logger.debug` please remove/comment them.
Since you have around 6 debug loggers within this pr.